circle.yml

```diff
@@ -1,59 +1,71 @@
+general:
+  artifacts:
+    - dashboard/test/ui
+    - dashboard/log
+    - pegasus/log
 machine:
   ruby:
     version: 2.2.3
+  node:
+    version: 0.12.0
   environment:
     RAILS_ENV: test
-    SPROCKETS_DERAILLEUR_WORKER_COUNT: 2 # CircleCI build VMs have a memory limit of 4G
+    RACK_ENV: test
+    SPROCKETS_DERAILLEUR_WORKER_COUNT: 2 # CircleCI build VMs have a memory limit of 8G
   services:
     - docker
 dependencies:
   cache_directories:
     - "apps/node_modules"
+    - "code-studio/node_modules"
     - "~/.rvm/gems/ruby-2.2.3"
   override:
     - sudo apt-get install libicu-dev
-    - bundle install
-    - "printf \"bundler_use_sudo: false\n\" > locals.yml"
-    - bundle exec rake install:pegasus
-    - npm install:
+    - bundle install --without ''
+    - 'echo "bundler_use_sudo: false" >> locals.yml'
```

Comment:
would it be cleaner to check in a `.circle.locals.yml` file and copy it here instead of doing a lot of echos?